src/Microsoft.Health.Fhir.Core/Messages/Search/SearchParameterDefinitionManagerInitialized.cs

```diff
@@ -0,0 +1,16 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using MediatR;
+
+namespace Microsoft.Health.Fhir.Core.Messages.Search
+{
+    /// <summary>
+    /// A notification that is raised when the SearchParameters are initialized
+    /// </summary>
+    public class SearchParameterDefinitionManagerInitialized : INotification
```

Comment:
Nit: if we're renaming `SearchParametersUpdated` to be `SearchParametersUpdatedNotification`, does it make sense for this to be `SearchParameterDefinitionManagerInitializedNotification`?

Reply:
(we can defer this to another PR if you'd like to get this merged)